delta/testkit/src/main/scala/ch/epfl/bluebrain/nexus/testkit/ce/CatsIOValues.scala

```diff
@@ -23,6 +23,8 @@ trait CatsIOValues extends CatsIOValuesLowPrio {
   implicit final class CatsIOValuesOps[A](private val io: IO[A]) {
     def accepted: A = io.unsafeRunSync()
 
+    def rejected2(implicit pos: source.Position): Throwable = rejectedWith[Throwable]
+
```

Comment:
`rejected2` should be renamed `rejected`

`rejected` should be renamed to something like `rejectedAssert`